laravel/resources/js/js-todo-list/index.js

```diff
@@ -0,0 +1,105 @@
+require('./bootstrap');
```

Comment:
![must](https://img.shields.io/badge/review-must-red.svg)
bootstrap は配置されていない、今回は使わないので、なくてもよいと思います。

Reply:
@mikankari 
bootstrap.js内でCSRFトークンを自動追加してると思ったので入れました！
https://readouble.com/laravel/7.x/ja/csrf.html
必要なさそうなら消しておきます

Reply:
その場合、正しいパスは１つ前かもしれません。

```suggestion
require('../bootstrap');
```
または下の import に揃えて次もよいかもしれません。
```suggestion
import '../bootstrap';
```

Reply:
その場合、正しいパスは１つ前かもしれません。

```suggestion
require('../bootstrap');
```
または下の import に揃えて次もよいかもしれません。
```suggestion
import '../bootstrap';
```